server/config/option.go

```diff
@@ -443,6 +443,17 @@ func (o *ScheduleOption) adjustScheduleCfg(persistentCfg *Config) {
 	}
 	scheduleCfg.Schedulers = append(scheduleCfg.Schedulers, restoredSchedulers...)
 	persistentCfg.Schedule.Schedulers = scheduleCfg.Schedulers
+
+	// the first time load from an old version
+	if persistentCfg.Schedule.DefaultStoreLimit ==
```

Comment:
Is there a test about reload from persist config?

Reply:
Well, I would like to add it.

Reply:
I found that the tests have already been added.

Reply:
@nolouch PTAL